CHANGELOG.md

```diff
@@ -26,6 +26,15 @@ Classify the change according to the following categories:
     ##### Removed
     ### Patches
 
+## v3.7.0
+### Minor Updates
+#### Changed
+- Default `Settings.solver_name` = `HiGHS`
+- See updates from REopt.jl v0.44.0: https://github.com/NREL/REopt.jl/releases/tag/v0.44.0
+- HiGHS, Cbc, and SCIP solvers use Big M notation constraints only in REopt.jl
+#### Deprecated
+- End-of-Life for v1 and v2 of the API for external/public interfacing from NREL servers
```

Comment:
@Bill-Becker should we include a bit more detail about what this means for users? E.g., Will users no longer be able to access results using v1 and v2 endpoints, or just will no longer be able to post to these endpoints?

Reply:
I made a discussion forum post about this, and added a link in both the API response error message and in the Changelog in https://github.com/NREL/REopt_API/pull/575/commits/089153d301053313bfcd01e030b1bd464a63cd67